opencodelists/models.py

```diff
@@ -3,6 +3,43 @@
 from django.utils.text import slugify
 
 
+class UserManager(BaseUserManager):
+    """
+    Custom user model manager where email is the unique identifiers
+    for authentication instead of usernames.
+    """
+
+    def create_user(self, username, email, password, **extra_fields):
+        """Create and save a User with the given email and password."""
+        if not email:
+            raise ValueError("The Email must be set")
+
+        org = Organisation.objects.first()
+        if not org:
+            org = Organisation.objects.create(
+                name="Data Lab", slug="datalab", url="https://ebmdatalab.net/"
```

Comment:
Nit (but I think it should be fixed) -- it's "DataLab" not "Data Lab".

Reply:
Fixed and rebased